paymentsheet/src/main/java/com/stripe/android/paymentsheet/navigation/PaymentSheetScreen.kt

```diff
@@ -34,13 +34,16 @@ import java.io.Closeable
 import com.stripe.android.R as PaymentsCoreR
 
 internal val verticalModeBottomContentPadding = 20.dp
+internal val horizontalModeWalletsDividerSpacing = 16.dp
+internal val verticalModeWalletsDividerSpacing = 24.dp
 
 internal sealed interface PaymentSheetScreen {
 
     val showsBuyButton: Boolean
     val showsContinueButton: Boolean
     val topContentPadding: Dp
     val bottomContentPadding: Dp
+    val walletsDividerSpacing: Dp
```

Comment:
@jaynewstrom-stripe not sure if this is the best approach for this, given that not that many screens show the wallets divider (so setting spacing for it feels a bit weird). What do you think?

Reply:
I think it makes sense 🤷